src/preamble.js

```diff
@@ -35,7 +35,16 @@ var wasmTable;
 #if USE_PTHREADS
 // For sending to workers.
 var wasmModule;
-#endif
+
+#if MODULARIZE
+// In modularize mode the wasmMemory and others are received in an onmessage, and that
```

Comment:
should this be "in pthreads mode" rather than "modularize mode"

Reply:
Fixing comment.